examples/Catena4430_Sensor/README.md

```diff
@@ -13,7 +15,7 @@ No problem, but wait for the red light to be out for 5 seconds.
 ### Typical File Content
 
 ```log
-Time,DevEUI,Raw,Vbat,Vsystem,Vbus,BootCount,T,RH,P,Light,P[0].delta,P[0].total,P[1].delta,P[1].total,Act[7],Act[6],Act[5],Act[4],Act[3],Act[2],Act[1],Act[0]
-2019-11-19T08:41:03Z,"0002cc010000046b","224afe6d1f7d421d523a1c1af15ea72d7b0062000000000000",4.13,,5.14,28,26.94,17.77,96923.00,98,0,0,0,0,,,,,,,,
-2019-11-19T08:51:26Z,"0002cc010000046b","224afe6f8efd3dd83a7b1c194d5ea432270066000000000000cdd4e61dfc53f47f6ff36d15",3.87,,3.65,28,25.30,19.59,96910.00,102,0,0,0,0,,,0.16,0.25,-0.28,-0.54,-0.10,-0.01
+Time,DevEUI,Raw,Vbat,Vbus,BootCount,T,RH,P,Light,P[0].delta,P[0].total,P[1].delta,P[1].total,Act[7],Act[6],Act[5],Act[4],Act[3],Act[2],Act[1],Act[0],NetWorkTime
+2019-11-19T08:41:03Z,"0002cc010000046b","224afe6d1fbf421d523a1c1af15ea72d7b0062000000000000",4.13,,5.14,28,26.94,17.77,96923.00,98,0,0,0,0,,,,,,,,True
+2019-11-19T08:51:26Z,"0002cc010000046b","224afe6f8eff3dd83a7b1c194d5ea432270066000000000000cdd4e61dfc53f47f6ff36d15",3.87,,3.65,28,25.30,19.59,96910.00,102,0,0,0,0,,,0.16,0.25,-0.28,-0.54,-0.10,-0.01,True
```

Comment:
For the port-based scheme, we can write `"*02message"` -- use the `*` to indicate that there's a port number at the start of the hex encoded message. That's pretty easy to decode.  Use port number 02 to indicate that the network time has not been received; port number 03 to indicate that the network time has been received. Leave the columns all as-is, don't add "NetworkTime" column.